### PR TITLE
Artsem_Klaprat: add 02.git.local

### PR DESCRIPTION
**Pull request by Geefzue**

### CODE REVIEW

File: Artsem_Klaprat/02.git.local.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ 2027  date > features-file1.txt
+ 2028  git add --all
+ 2029  git commit -m "1st commit to features"
+ 2030  date > features-file2.txt
+ 2031  git add --all
+ 2032  git commit -m "2st commit to features"
+ 2033  git log --oneline 
+ 2034  git rebase -i HEAD~N
+ 2035  git rebase -i HEAD
+ 2036* git log --oneline
```
Comment (pluhin) [inline Artsem_Klaprat/02.git.local.md]: Can you provide output?

File: Artsem_Klaprat/02.git.local.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ 2037  git rebase -i HEAD features/do_one 
+ 2038  git log --oneline 
+ 2039  git checkout -b hotfix/we_gonna_die
+ 2040  date > hot-file1.txt
+ 2041  git add --all
+ 2042  git commit -m "1st commit to features"
+ 2043  ls
+ 2044  git branch -a
+ 2045  git checkout master
+ 2046  git log --oneline
```
Comment (pluhin) [inline Artsem_Klaprat/02.git.local.md]: Can you provide output?

File: Artsem_Klaprat/02.git.local.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+ 2047  git merge features/do_one
+ 2048  git checkout -b hotfix/we_gonna_di
+ 2049  git log --oneline
```
Comment (pluhin) [inline Artsem_Klaprat/02.git.local.md]: Can you provide output?

File: Artsem_Klaprat/02.git.local.md (hunk 2 above)
Comment (Geefzue) [reply Artsem_Klaprat/02.git.local.md]: it branch -a
  dev
  features/do_one
  hotfix/we_gonna_di
  hotfix/we_gonna_die
* master
git checkout master
Уже на «master»

git log --oneline
915986a (HEAD -> master, hotfix/we_gonna_di, features/do_one) 2st commit to features
e0847cc 1st commit to features
fef40e2 (dev) 2st commit to dev
8f33ce2 1st commit to dev
e4acb39 2st commit to master
7de246f 1st commit to master

File: Artsem_Klaprat/02.git.local.md (hunk 3 above)
Comment (Geefzue) [reply Artsem_Klaprat/02.git.local.md]: git merge features/do_one
Уже обновлено.
git checkout -b hotfix/we_gonna_di
fatal: Ветка с именем «hotfix/we_gonna_di» уже существует.

**Pull request merged**